6_handbook/20_application_tuning.txt

```diff
@@ -204,15 +204,25 @@ mount /var/rudder/cfengine-community/state
 
 ==== Rsyslog
 
-On Rudder policy servers (root or relay), when managing a large number of nodes,
-you can experience issues with rsyslog. This happens because Rudder uses TCP by
-default for sending reports to rsyslog, which implies the system has to keep track of
+If you upgraded your Rudder server from a pre-3.1 version or if you have changed the 
```

Comment:
Ahhhh, I understand this now I've re-read the section from 12_upgrade/05_caution.txt!

This needs explaining again here, maybe like this:

> Rudder 3.1 and later use UDP by default for reporting, but if you upgraded your server from a previous version, you will keep the previous setting which is TCP.